File: ScpProject/UI/ViewModels/Athlete/CompleteRegistration.cs
namespace Controllers.ViewModels.Athlete
{
    public class CompleteRegistration
    {
        public string UserName { get; set; }
        public string Password { get; set; }
        public string ConfirmPassword { get; set; }
        public string EmailValidationToken  { get; set; }
        public int AthleteId { get; set; }
    }
}

File: ScpProject/StrengthConApp/src/app/Models/Athlete/CompleteRegistration.ts
export class CompleteRegistration {
    public UserName: string;
    public Password: string;
    public ConfirmPassword: string;
    public EmailValidationToken: string;
    public AthleteId: number;
}
export class CompleteCoachRegistration {
    public UserName: string;
    public Password: string;
    public ConfirmPassword: string;
    public EmailValidationToken: string;
    public UserId: number;
}
export class OneTime {
    public UserName: string;
    public Password: string;
    public ConfirmPassword: string;
    public EmailValidationToken: string;
    public UserId: number;
    public OrgName: string;
}